src/app/test/dot-workflow-service.mock.ts

```diff
@@ -1,7 +1,10 @@
 import { Observable } from 'rxjs/Observable';
 import { DotWorkflowAction } from '../shared/models/dot-workflow-action/dot-workflow-action.model';
+import { DotWorkflow } from '../shared/models/dot-workflow/dot-workflow.model';
+import * as fp from 'lodash/fp';
+import cloneDeep from 'lodash/fp/cloneDeep';
```

Comment:
In line 4, you import all from lodash, so there's no need to import this on line 5